opm/input/eclipse/Schedule/Well/Well.cpp

```diff
@@ -2038,6 +2052,7 @@ bool Well::cmp_structure(const Well& other) const
         && (this->udq_undefined == other.udq_undefined)
         && (this->getPreferredPhase() == other.getPreferredPhase()) // wellType()
         && (this->getEfficiencyFactor() == other.getEfficiencyFactor())
```

Comment:
It does not matter for the current usage since the argument is false by default, but it might be better to compare the value 
`this->efficiency_factor == other.efficiency_factor` now, since the function `getEfficiencyFactor()` uses `use_efficiency_in_network` and it is a little more complex function now.

Reply:
> It does not matter for the current usage since the argument is false by default, but it might be better to compare the value `this->efficiency_factor == other.efficiency_factor` now, since the function `getEfficiencyFactor()` uses `use_efficiency_in_network` and it is a little more complex function now.

Arguably, two wells aren't equal if their `use_efficiency_in_network` differ so I think it actually makes sense to continue to use `getEfficiencyFactor()` for this criterion.

Reply:
Sure. 

In the current form of the function `double getEfficiencyFactor(bool network)`, it is fine and it is equivalent to compare with the  `efficiency_factor`.  But I do feel that it is safer (and slightly faster) to compare with the member variable directly. 

In a more general setting, for example, the function `double getEfficiencyFactor(bool network)` evolves somehow it  might not reflect the value of `efficiency_factor`, then a bug can be introduced. Or we might need to compare both `getEfficiencyFactor(true)` and `getEfficiencyFactor(false)`. 

As said, I think it does not make much difference for the current code, but I do think compare with the member variable itself is safer generally.